gulpfile.js

```diff
@@ -59,31 +85,30 @@ gulp.task('package-resources', ['default'], () => {
   ]);
 });
 
-const browserifyCb = (cb) => {
-  return (err, stdout, stderr) => {
-    console.log(stdout);
-    console.error(stderr);
-    cb(err);
-  };
-};
-
-gulp.task('package-render', ['default'], (cb) => {
-  exec('browserify --node -e build/index.js -o ./package/index.js --im', browserifyCb(cb));
+gulp.task('package-render', ['default', 'package-init'], () => {
+  browserify('build/index.js', browserifyOpts)
```

Comment:
Just a heads up, this isn't exactly the same as before. There's a difference between render and main - render has `--igv=false` omitted (*see lines 71 and 75 of original*)

Interestingly, the [default](https://github.com/substack/node-browserify#usage) is not even a boolean, but `__filename,__dirname,process,Buffer,global`.